.github/workflows/ci.yml

```diff
@@ -157,23 +182,6 @@ jobs:
             next-version: '15'
 
     steps:
-      # Skip integration tests from fork PRs to prevent secret exfiltration
```

Comment:
This was individually running for integration tests, but we run should place it on the root, before running the setup for Turborepo + Blacksmith.